### ISSUE TITLE
Issue with creating azure external stage -- azure_sas_token is mangled when passed in as variable.

### ISSUE THREAD

**Issue by holtalanm** [Snowflake-Labs/schemachange#99]
I have an 'always' script that is responsible for replacing an external stage with a new SAS token on each deployment.  The stage url and sas token are passed in as variables, and they get passed in correctly, but upon execution, the sas token seems to be getting double-encoded.

Below is my script:

```sql
use schema "PUBLIC";

-- will always run, updating SAS token for stage on each deploy.
create or replace stage foo_stage_extern
    url = '{{foo_stage_url}}'
    credentials=(azure_sas_token='{{foo_stage_sas}}')
    file_format = foo_csv;
```

The log when running shows the sas token was passed into schemachange correctly:

```
Using variables {
  'foo_stage_sas': '?sv=2020-02-10&ss=bf&srt=sco&sp=rwdlac&se=2031-11-13T15:50:49Z&st=2021-11-15T15:50:49Z&spr=https&sig=signature', 
  'foo_stage_url': 'azure://storageaccountname.blob.core.usgovcloudapi.net/foo-staged-files'
}
```

But the actual executed sql appears to be encoding the variable (note the `&amp;` in place of the `&` for the query string):

```sql
create or replace stage foo_stage_extern

    url = 'azure://storageaccountname.blob.core.usgovcloudapi.net/foo-staged-files'

    credentials=(azure_sas_token='?sv=2020-02-10&amp;ss=bf&amp;srt=sco&amp;sp=rwdlac&amp;se=2031-11-13T15:50:49Z&amp;st=2021-11-15T15:50:49Z&amp;spr=https&amp;sig=signature')

    file_format = foo_csv
```

**Comment by holtalanm** [comment]
Looks like schemachange is using automatic html escaping with Jinja, which is not the default behavior of Jinja according to the docs.


If anyone else runs into this, see:  https://jinja.palletsprojects.com/en/3.0.x/templates/#working-with-automatic-escaping

I was able to fix this with:

```sql
use schema "PUBLIC";

-- will always run, updating SAS token for stage on each deploy.
create or replace stage foo_stage_extern
    url = '{{foo_stage_url|safe}}'
    credentials=(azure_sas_token='{{foo_stage_sas|safe}}')
    file_format = foo_csv;

```

**Comment by holtalanm** [comment]
Looks like this was bad behavior within v3.3.2

Also looks to be fixed in v3.3.3 according to #92 

see: https://github.com/Snowflake-Labs/schemachange/issues/92#issuecomment-964522116

and: https://github.com/Snowflake-Labs/schemachange/issues/92#issuecomment-964563914

**Comment by sfc-gh-jhansen** [comment]
Yep, this has been fixed and published in v3.3.3.
